src/main/java/task8/Coordinates.java

```diff
@@ -0,0 +1,12 @@
+package task8;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Coordinates {
+    @JsonProperty("lon")
+    public String longitude;
+    @JsonProperty("lat")
+    public String latitude;
```

Comment:
It would be better to make fields private and to implement getters and setters. It will allow using extend these classes if needed and use them as business objects in big framework.